Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```
@@ -129,6 +129,8 @@
   - type: WiresPanel
   - type: Physics
     bodyType: Dynamic
+    angularDamping: 10 #sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)